result/config_log.txt
Time: 2022-09-26 11:32:45.048533 
N_HIDDEN_LAYER: 8 
N_HIDDEN: 64 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 64
STEP:100
GAMMA:0.1
DROPOUT:0.2
First Train LOSS: 178.43592834472656 
First Valid LOSS: 181.3103790283203 
Last Train LOSS: 0.19655737280845642 
Last Valid LOSS: 0.28266122937202454 

Time: 2022-09-26 11:37:52.214397 
N_HIDDEN_LAYER: 8 
N_HIDDEN: 80 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 64
STEP:100
GAMMA:0.1
DROPOUT:0.2
First Train LOSS: 83.6803207397461 
First Valid LOSS: 83.85450744628906 
Last Train LOSS: 0.111217200756073 
Last Valid LOSS: 0.18208728730678558 

Time: 2022-09-26 12:49:08.786435 
N_HIDDEN_LAYER: 8 
N_HIDDEN: 80 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 128
STEP:100
GAMMA:0.1
DROPOUT:0.2
First Train LOSS: 1046.217529296875 
First Valid LOSS: 1029.0936279296875 
Last Train LOSS: 0.1282563954591751 
Last Valid LOSS: 0.17779545485973358 

Time: 2022-09-26 13:19:54.698688 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 128 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 128
STEP:100
GAMMA:0.5
DROPOUT:0.1
First Train LOSS: 790.8984985351562 
First Valid LOSS: 794.1964721679688 
Last Train LOSS: 0.1126529797911644 
Last Valid LOSS: 0.14127232134342194 

Time: 2022-09-26 13:43:40.720493 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 192 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 128
STEP:100
GAMMA:0.5
DROPOUT:0.1
First Train LOSS: 436.2999267578125 
First Valid LOSS: 440.49029541015625 
Last Train LOSS: 0.05989772453904152 
Last Valid LOSS: 0.09857290238142014 

Time: 2023-02-15 21:49:43.640555 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 192 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 128
STEP:100
GAMMA:0.5
DROPOUT:0.1
First Train LOSS: 1964.136474609375 
First Valid LOSS: 1963.523193359375 
Last Train LOSS: 0.035262368619441986 
Last Valid LOSS: 0.09999934583902359 

Time: 2023-02-15 21:51:58.343438 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 192 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 128
STEP:100
GAMMA:0.5
DROPOUT:0.1
First Train LOSS: 1674.2657470703125 
First Valid LOSS: 1669.5672607421875 
Last Train LOSS: 0.02142300270497799 
Last Valid LOSS: 0.09999200701713562 

Time: 2023-02-15 21:52:41.860057 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 192 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 128
STEP:100
GAMMA:0.5
DROPOUT:0.1
First Train LOSS: 1559.12451171875 
First Valid LOSS: 1549.5946044921875 
Last Train LOSS: 0.035809990018606186 
Last Valid LOSS: 0.09999807178974152 

Time: 2023-02-15 21:55:39.483589 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 192 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 128
STEP:100
GAMMA:0.5
DROPOUT:0.1
First Train LOSS: 1637.6185302734375 
First Valid LOSS: 1637.33984375 
Last Train LOSS: 0.04078301414847374 
Last Valid LOSS: 0.09993521869182587 

Time: 2023-02-15 21:56:47.841386 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 192 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 128
STEP:100
GAMMA:0.5
DROPOUT:0.1
First Train LOSS: 1837.9534912109375 
First Valid LOSS: 1837.1387939453125 
Last Train LOSS: 0.03818351402878761 
Last Valid LOSS: 0.09999382495880127 

Time: 2023-02-15 21:58:21.180767 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 192 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 128
STEP:100
GAMMA:0.5
DROPOUT:0.1
First Train LOSS: 1587.2071533203125 
First Valid LOSS: 1582.7965087890625 
Last Train LOSS: 0.042850688099861145 
Last Valid LOSS: 0.09998340159654617 

Time: 2023-02-15 22:05:53.608831 
N_HIDDEN_LAYER: 4 
N_HIDDEN: 32 
EPOCH: 1000 
LR: 0.1 
BATCH_SIZE: 127
STEP:200
GAMMA:0.5
DROPOUT:0.5
First Train LOSS: 1545.937744140625 
First Valid LOSS: 1548.8328857421875 
Last Train LOSS: 0.11281612515449524 
Last Valid LOSS: 0.22472308576107025 

Time: 2023-02-15 22:08:30.925788 
N_HIDDEN_LAYER: 4 
N_HIDDEN: 32 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.5
DROPOUT:0.5
First Train LOSS: 1619.7222900390625 
First Valid LOSS: 1623.549072265625 
Last Train LOSS: 0.0030395190697163343 
Last Valid LOSS: 0.3098287880420685 

Time: 2023-02-15 22:16:13.482296 
N_HIDDEN_LAYER: 4 
N_HIDDEN: 128 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:1000
GAMMA:0.5
DROPOUT:0.5
First Train LOSS: 1593.85205078125 
First Valid LOSS: 1594.395263671875 
Last Train LOSS: 0.00024149217642843723 
Last Valid LOSS: 0.5413899421691895 

Time: 2023-02-15 22:22:20.962440 
N_HIDDEN_LAYER: 3 
N_HIDDEN: 64 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1757.7518310546875 
First Valid LOSS: 1759.8115234375 
Last Train LOSS: 0.1157064363360405 
Last Valid LOSS: 0.19838769733905792 

Time: 2023-02-15 22:24:43.949990 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1765.26123046875 
First Valid LOSS: 1766.7528076171875 
Last Train LOSS: 0.029341785237193108 
Last Valid LOSS: 0.19969752430915833 

Time: 2023-02-15 22:24:53.302665 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1747.646240234375 
First Valid LOSS: 1744.710693359375 
Last Train LOSS: 0.06724897027015686 
Last Valid LOSS: 0.19982391595840454 

Time: 2023-02-15 22:25:51.146952 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1616.1190185546875 
First Valid LOSS: 1617.990234375 
Last Train LOSS: 0.08301360160112381 
Last Valid LOSS: 0.19798849523067474 

Time: 2023-02-15 22:48:16.009232 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1578.724609375 
First Valid LOSS: 1579.4495849609375 
Last Train LOSS: 0.10680343955755234 
Last Valid LOSS: 0.19682565331459045 

Time: 2023-02-28 23:49:22.881372 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1713.583251953125 
First Valid LOSS: 1714.9158935546875 
Last Train LOSS: 0.09081794321537018 
Last Valid LOSS: 0.19819822907447815 

Time: 2023-03-01 19:54:16.271288 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1773.2967529296875 
First Valid LOSS: 1775.8572998046875 
Last Train LOSS: 0.07335560768842697 
Last Valid LOSS: 0.19212749600410461 

Time: 2023-03-09 10:00:36.460193 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1817.533203125 
First Valid LOSS: 1816.5628662109375 
Last Train LOSS: 0.07265355437994003 
Last Valid LOSS: 0.19815202057361603 

Time: 2023-03-09 10:02:32.735953 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1632.400634765625 
First Valid LOSS: 1635.0511474609375 
Last Train LOSS: 0.08648312836885452 
Last Valid LOSS: 0.1986704021692276 

Time: 2023-03-09 10:03:53.191954 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1851.6771240234375 
First Valid LOSS: 1855.1258544921875 
Last Train LOSS: 0.07772822678089142 
Last Valid LOSS: 0.1993350237607956 

Time: 2023-03-09 10:06:10.903018 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1719.4571533203125 
First Valid LOSS: 1718.8922119140625 
Last Train LOSS: 0.11748494952917099 
Last Valid LOSS: 0.19897429645061493 

Time: 2023-03-09 10:11:40.649996 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1746.5882568359375 
First Valid LOSS: 1745.6055908203125 
Last Train LOSS: 0.07880502194166183 
Last Valid LOSS: 0.19835400581359863 

Time: 2023-03-09 10:11:54.622105 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1813.293212890625 
First Valid LOSS: 1814.4808349609375 
Last Train LOSS: 0.07441750168800354 
Last Valid LOSS: 0.1986948698759079 

Time: 2023-03-09 10:12:26.794759 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1702.9510498046875 
First Valid LOSS: 1703.000244140625 
Last Train LOSS: 0.06000519171357155 
Last Valid LOSS: 0.19932617247104645 

Time: 2023-03-09 10:13:12.834315 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1639.0814208984375 
First Valid LOSS: 1640.8111572265625 
Last Train LOSS: 0.06542815268039703 
Last Valid LOSS: 0.19550055265426636 

Time: 2023-03-09 10:15:08.472103 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1772.8607177734375 
First Valid LOSS: 1771.299072265625 
Last Train LOSS: 0.08389181643724442 
Last Valid LOSS: 0.19370177388191223 

Time: 2023-03-09 10:16:16.431205 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1800.306640625 
First Valid LOSS: 1802.6563720703125 
Last Train LOSS: 0.10019683092832565 
Last Valid LOSS: 0.19285275042057037 

Time: 2023-03-09 10:16:34.667608 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1701.2503662109375 
First Valid LOSS: 1700.5955810546875 
Last Train LOSS: 0.050948355346918106 
Last Valid LOSS: 0.19997552037239075 

Time: 2023-03-09 10:19:05.292255 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1842.4647216796875 
First Valid LOSS: 1843.7962646484375 
Last Train LOSS: 0.08809728175401688 
Last Valid LOSS: 0.19926799833774567 

Time: 2023-03-09 10:19:53.633230 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1820.5792236328125 
First Valid LOSS: 1821.6942138671875 
Last Train LOSS: 0.1216229721903801 
Last Valid LOSS: 0.19831791520118713 

Time: 2023-03-09 10:20:19.271207 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1689.298828125 
First Valid LOSS: 1693.7076416015625 
Last Train LOSS: 0.055368490517139435 
Last Valid LOSS: 0.19783174991607666 

Time: 2023-03-09 10:20:40.733461 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1608.7288818359375 
First Valid LOSS: 1609.609619140625 
Last Train LOSS: 0.07263512164354324 
Last Valid LOSS: 0.19939681887626648 

Time: 2023-03-09 10:21:14.509994 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1462.496826171875 
First Valid LOSS: 1465.485595703125 
Last Train LOSS: 0.14085407555103302 
Last Valid LOSS: 0.19926124811172485 

Time: 2023-03-09 10:22:16.975539 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1688.52294921875 
First Valid LOSS: 1687.36767578125 
Last Train LOSS: 0.07294756174087524 
Last Valid LOSS: 0.1986662745475769 

Time: 2023-03-09 10:30:23.313836 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1628.539306640625 
First Valid LOSS: 1629.146240234375 
Last Train LOSS: 0.11837857216596603 
Last Valid LOSS: 0.19910433888435364 

Time: 2023-03-09 10:31:05.419532 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1687.43408203125 
First Valid LOSS: 1691.5511474609375 
Last Train LOSS: 0.03692702576518059 
Last Valid LOSS: 0.1995457112789154 

Time: 2023-03-09 10:31:36.161732 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1669.4417724609375 
First Valid LOSS: 1673.32958984375 
Last Train LOSS: 0.08475381880998611 
Last Valid LOSS: 0.19986487925052643 

Time: 2023-03-09 10:41:53.036527 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1671.565673828125 
First Valid LOSS: 1665.76416015625 
Last Train LOSS: 0.0648060068488121 
Last Valid LOSS: 0.19457687437534332 

Time: 2023-03-09 10:42:27.153728 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1661.6881103515625 
First Valid LOSS: 1664.3541259765625 
Last Train LOSS: 0.06103099137544632 
Last Valid LOSS: 0.19615599513053894 

Time: 2023-03-09 10:43:00.699560 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1842.19580078125 
First Valid LOSS: 1847.678466796875 
Last Train LOSS: 0.01958240568637848 
Last Valid LOSS: 0.19749628007411957 

Time: 2023-03-09 10:44:20.484780 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1735.2998046875 
First Valid LOSS: 1730.997802734375 
Last Train LOSS: 0.1146610826253891 
Last Valid LOSS: 0.1990613490343094 

Time: 2023-03-09 10:44:59.482490 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1654.50341796875 
First Valid LOSS: 1654.6405029296875 
Last Train LOSS: 0.02591826021671295 
Last Valid LOSS: 0.19893959164619446 

Time: 2023-03-09 10:46:22.464958 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1756.8582763671875 
First Valid LOSS: 1758.1533203125 
Last Train LOSS: 0.08483986556529999 
Last Valid LOSS: 0.1979147046804428 

Time: 2023-03-09 10:47:58.026144 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1718.359619140625 
First Valid LOSS: 1721.1278076171875 
Last Train LOSS: 0.05330884829163551 
Last Valid LOSS: 0.19669796526432037 

Time: 2023-03-09 10:48:10.449152 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1663.8387451171875 
First Valid LOSS: 1664.0689697265625 
Last Train LOSS: 0.09239763021469116 
Last Valid LOSS: 0.19924496114253998 

Time: 2023-03-09 10:49:01.222611 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1701.9376220703125 
First Valid LOSS: 1703.87060546875 
Last Train LOSS: 0.09925152361392975 
Last Valid LOSS: 0.19796177744865417 

Time: 2023-03-09 10:49:20.727551 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1757.7457275390625 
First Valid LOSS: 1757.5113525390625 
Last Train LOSS: 0.13673195242881775 
Last Valid LOSS: 0.19978515803813934 

Time: 2023-03-09 10:51:40.040798 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1791.322021484375 
First Valid LOSS: 1777.08251953125 
Last Train LOSS: 0.0854523554444313 
Last Valid LOSS: 0.24639153480529785 

Time: 2023-03-09 10:54:13.360613 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1837.290283203125 
First Valid LOSS: 1818.3900146484375 
Last Train LOSS: 0.0707993358373642 
Last Valid LOSS: 0.2492092102766037 

Time: 2023-03-09 11:06:39.873411 
N_HIDDEN_LAYER: 3 
N_HIDDEN: 32 
EPOCH: 2000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 4645.55859375 
First Valid LOSS: 4638.08642578125 
Last Train LOSS: 0.6916050314903259 
Last Valid LOSS: 2.9043264389038086 

Time: 2023-03-09 11:07:36.963675 
N_HIDDEN_LAYER: 3 
N_HIDDEN: 32 
EPOCH: 2000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 4866.82958984375 
First Valid LOSS: 4863.810546875 
Last Train LOSS: 0.48614731431007385 
Last Valid LOSS: 5.205506324768066 

Time: 2023-03-09 11:08:47.838675 
N_HIDDEN_LAYER: 3 
N_HIDDEN: 32 
EPOCH: 2000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 4759.109375 
First Valid LOSS: 4766.1357421875 
Last Train LOSS: 0.578741729259491 
Last Valid LOSS: 9.532249450683594 

Time: 2023-03-09 11:09:49.007669 
N_HIDDEN_LAYER: 3 
N_HIDDEN: 32 
EPOCH: 2000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 4707.4658203125 
First Valid LOSS: 4718.5556640625 
Last Train LOSS: 0.5700249671936035 
Last Valid LOSS: 6.3177666664123535 

Time: 2023-03-09 11:25:38.370928 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1811.791748046875 
First Valid LOSS: 1812.63134765625 
Last Train LOSS: 0.15485316514968872 
Last Valid LOSS: 0.24939407408237457 

Time: 2023-03-09 11:26:40.366132 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1529.05029296875 
First Valid LOSS: 1531.638916015625 
Last Train LOSS: 0.12541930377483368 
Last Valid LOSS: 0.24977493286132812 

Time: 2023-03-09 11:28:50.710142 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1735.114990234375 
First Valid LOSS: 1724.678466796875 
Last Train LOSS: 0.04855027422308922 
Last Valid LOSS: 0.24926239252090454 

Time: 2023-03-09 11:31:13.838271 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1767.2061767578125 
First Valid LOSS: 1759.6365966796875 
Last Train LOSS: 0.015516752377152443 
Last Valid LOSS: 0.8151118159294128 

Time: 2023-03-09 11:31:50.215949 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1746.3460693359375 
First Valid LOSS: 1736.6854248046875 
Last Train LOSS: 0.051380034536123276 
Last Valid LOSS: 0.24376818537712097 

Time: 2023-03-09 11:35:17.069791 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1700.3255615234375 
First Valid LOSS: 1717.938720703125 
Last Train LOSS: 0.010565018281340599 
Last Valid LOSS: 0.4498770833015442 

Time: 2023-03-11 09:59:38.404677 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1709.6806640625 
First Valid LOSS: 1727.9993896484375 
Last Train LOSS: 0.14378607273101807 
Last Valid LOSS: 0.44281062483787537 

Time: 2023-03-11 10:00:37.505538 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1632.9541015625 
First Valid LOSS: 1653.373046875 
Last Train LOSS: 0.04202017933130264 
Last Valid LOSS: 0.44960618019104004 

Time: 2023-03-11 10:04:29.161643 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 10000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1707.9638671875 
First Valid LOSS: 1699.9091796875 
Last Train LOSS: 0.14926548302173615 
Last Valid LOSS: 0.4491308033466339 

Time: 2023-03-11 10:30:34.686546 
N_HIDDEN_LAYER: 4 
N_HIDDEN: 8 
EPOCH: 2000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1616.5333251953125 
First Valid LOSS: 1607.9954833984375 
Last Train LOSS: 3.4810657501220703 
Last Valid LOSS: 7.654297351837158 

Time: 2023-03-11 10:31:44.263621 
N_HIDDEN_LAYER: 4 
N_HIDDEN: 20 
EPOCH: 2000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1591.6767578125 
First Valid LOSS: 1583.487548828125 
Last Train LOSS: 0.03371363505721092 
Last Valid LOSS: 0.6431558728218079 

Time: 2023-03-11 10:37:40.071642 
N_HIDDEN_LAYER: 4 
N_HIDDEN: 96 
EPOCH: 2000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1632.27197265625 
First Valid LOSS: 1624.1591796875 
Last Train LOSS: 0.010996674187481403 
Last Valid LOSS: 0.6088393926620483 

Time: 2023-03-11 10:41:23.350606 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 2000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1540.9154052734375 
First Valid LOSS: 1530.754638671875 
Last Train LOSS: 0.011318179778754711 
Last Valid LOSS: 0.36243149638175964 

Time: 2023-03-11 10:52:42.750949 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 2000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1783.1815185546875 
First Valid LOSS: 1773.612060546875 
Last Train LOSS: 0.09210776537656784 
Last Valid LOSS: 0.459032267332077 

Time: 2023-03-11 10:53:24.519539 
N_HIDDEN_LAYER: 2 
N_HIDDEN: 96 
EPOCH: 5000 
LR: 0.1 
BATCH_SIZE: 127
STEP:2000
GAMMA:0.9
DROPOUT:0.5
First Train LOSS: 1662.966796875 
First Valid LOSS: 1655.2015380859375 

config.py
import datetime
import os.path

import torch

ABSPATH = os.path.split(os.path.realpath(__file__))[0]


class Config:
    # Hyper parameters
    N_INPUT = 4
    N_HIDDEN_LAYER = 2
    N_HIDDEN = 96
    N_OUTPUT = 3
    ACTIVATION = torch.tanh
    B_INIT = -0.2  # use a bad bias constant initializer
    EPOCH = 2e3
    LR = 1e-1
    BATCH_SIZE = 127
    STEP = 500
    GAMMA = 0.1
    DROPOUT = 0.5
    EVOLUTION_MAX_PROC = 12

    def __init__(self):
        self.EPOCH = int(self.EPOCH)

    def save(self, train_loss, valid_loss):
        with open(os.path.join(ABSPATH, 'result', 'config_log.txt'), 'a+') as f:
            f.write(f'Time: {datetime.datetime.now()} \n')
            f.write(f"N_HIDDEN_LAYER: {self.N_HIDDEN_LAYER} \n")
            f.write(f"N_HIDDEN: {self.N_HIDDEN} \n")
            f.write(f"EPOCH: {self.EPOCH} \n")
            f.write(f"LR: {self.LR} \n")
            f.write(f"BATCH_SIZE: {self.BATCH_SIZE}\n")
            f.write(f"STEP:{self.STEP}\n")
            f.write(f"GAMMA:{self.GAMMA}\n")
            f.write(f"DROPOUT:{self.DROPOUT}\n")
            f.write(f"First Train LOSS: {train_loss[0]} \n")
            f.write(f"First Valid LOSS: {valid_loss[0]} \n")
            f.write(f"Last Train LOSS: {train_loss[-1]} \n")
            f.write(f"Last Valid LOSS: {valid_loss[-1]} \n\n")
